src/Effects.js

```diff
@@ -1445,6 +1445,40 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.squiggles = {
+      points: [],
+      dotSpacing: 3,
+      amplitude: 40,
+      period: 400,
+      dotRadius: 7,
+      numSquiggles: 8,
+      init: function () {
+        this.points = [];
+        p5.noStroke();
+        p5.angleMode(p5.DEGREES);
+        let numPoints = p5.height / this.dotSpacing;
+        for (var i = 0; i < numPoints; i++) {
+          this.points.push({
+            x: 0,
+            y: i * this.dotSpacing,
+            theta: 360 / this.period * i * this.dotSpacing,
```

Comment:
What is the theta?

Reply:
Theta is the current angle of the sine curve for each dot